.github/workflows/run_secret_scan.yaml

```diff
@@ -0,0 +1,26 @@
+# Purpose: Run a secret scanner against the repo.
+
+name: Run Secret Scan
+
+on:
+  push:
+    branches:
+      - 840-research-third-party-secret-scanners
```

Comment:
We used a more generic branch like *-secret-scanner to accommodate future branches and not having to manage pulling specific out of pull requests.

Reply:
Good catch, I had forgotten to remove this branch, which is only necessary for testing.  Removed.